descriptors/ModuleDescriptor-template.json

```diff
@@ -2,9 +2,71 @@
   "id": "${artifactId}-${version}",
   "name": "Invoice CRUD module",
   "provides": [
+    {
+      "id": "invoice-storage.invoice",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/invoice-storage/invoices",
+          "permissionsRequired": ["invoice.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/invoice-storage/invoices",
+          "permissionsRequired": ["invoice.item.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/invoice-storage/invoices/{id}",
+          "permissionsRequired": ["invoice.item.get"]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/invoice-storage/invoices/{id}",
+          "permissionsRequired": ["invoice.item.put"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/invoice-storage/invoices/{id}",
+          "permissionsRequired": ["invoice.item.delete"]
+        }
+      ]
+    },
+    {
+      "id": "invoice-storage.invoice-line",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/invoice-storage/invoice-lines",
+          "permissionsRequired": ["invoice-lines.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/invoice-storage/invoice-lines",
+          "permissionsRequired": ["invoice-lines.item.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/invoice-storage/invoice-lines/{id}",
+          "permissionsRequired": ["invoice-lines.item.get"]
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/invoice-storage/invoice-lines/{id}",
+          "permissionsRequired": ["invoice-lines.item.put"]
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/invoice-storage/invoice-lines/{id}",
+          "permissionsRequired": ["invoice-lines.item.delete"]
+        }
+      ]
+    },
     {
       "id": "_tenant",
-      "version": "1.2",
+      "version": "1.0",
```

Comment:
please use 1.2.  If we ever want to be able to load reference/sample data via the new preferred mechanism (tenant API), we'll need at 1.2